.github/workflows/check.yml

```diff
@@ -55,4 +55,4 @@ jobs:
           cargo check --all-features
       - name: Run Tests
         run: |
-          cargo test --release --all-features
+          cargo test --all-features
```

Comment:
Why removing release?

Reply:
The time spent drops from 1 hour to 25 minutes.

Reply:
All the other commands are in debug, so doing the final one in release added a lot of extra time.